apps/consumer-client/src/pages/examples/gpc-proof.tsx

```diff
@@ -108,6 +126,26 @@ export default function Page(): JSX.Element {
           }}
         />
         <br />
+        Prescribed entries (or empty for none):
+        <textarea
+          cols={45}
+          rows={5}
+          value={prescribedEntries}
+          onChange={(e): void => {
+            setPrescribedEntries(e.target.value);
+          }}
+        />
+        <br />
+        Prescribed signers' public keys (or empty for none):
```

Comment:
Prescribed public keys to filter which PODs user can select.  These keys must be revealed in the configuration.  (Use empty for none.)